tests/quantization/test_configs.py

```diff
@@ -55,6 +56,8 @@ class ModelPair:
 ]
 
 
+@pytest.mark.skipif(torch.cuda.get_device_capability() < (8, 0),
+                    reason="skip for T4s")
```

Comment:
Can you update this to:

"Skip for t4, because `marlin` requires compute capability 8.0"

Reply:
sure